src/Kanbas/Courses/Quizzes/Preview/QuestionLinks/index.tsx

```diff
@@ -1,6 +1,62 @@
-function QuizPreviewQuestionLinks() {
+import { Button } from "react-bootstrap";
+import { MdOutlineEdit } from "react-icons/md";
+import { Link, useParams } from "react-router-dom";
+import { GoQuestion } from "react-icons/go";
+import "./index.css";
+import { useDispatch, useSelector } from "react-redux";
+import { KanbasState } from "../../../../store";
+import { IKanbasQuizQuestion } from "../../../../store/interfaces/quizzes";
+import { FaFlag } from "react-icons/fa";
+import { setActiveQuestion } from "../../reducer";
+
+function QuizPreviewQuestionLinks({ scrollToQuestion }:
+    { scrollToQuestion?: any }) {
+
+    const { courseId, quizId } = useParams();
+    const dispatch = useDispatch();
+
+    const flaggedQuestions = useSelector(
```

Comment:
This doesn't need to be in redux state, as this is only applicable for the quiz preview.
Instead, use another redux state for preview components or use useState and pass around the state via params.